src/ontology/config/property-map.sssom.tsv

```diff
@@ -7,3 +7,4 @@ http://www.ebi.ac.uk/efo/reason_for_obsolescence	rdfs:comment
 http://www.ebi.ac.uk/efo/alternative_term	oboInOwl:hasExactSynonym
 http://purl.obolibrary.org/obo/ECO_0000218	oboInOwl:source
 http://purl.obolibrary.org/obo/IAO_0000118	oboInOwl:hasExactSynonym
+http://purl.obolibrary.org/obo/OMO_0003012	http://purl.obolibrary.org/obo/mondo#ABBREVIATION
```

Comment:
## `synonym_type` from sources
- [ ] @twhetzel @matentzn Review and assist with questions

@twhetzel suggested I add `OMO:0003012` (abbreviation) to `properties.txt`.

However I had the idea to convert it to `MONDO:ABBREVIATION`. So I added an entry to `property-map.sssom.tsv`.

However it's not working correctly. I got some strange behavior in my robot conversions. One time, I got:
```
NEW IRI ERROR failed to generate an IRI for ''.
For details see: http://robot.obolibrary.org/rename#new-iri-error
```

Anyway here's what it looks like afterwards. I got an annotation property declaration for `MONDO:ABBREVIATION`, no declaration for `OMOM:0003012`, but 0 usages of `MONDO:ABBREVIATION`; all the axioms are still using `OMOM:0003012`.

<details><summary>Details</summary>
<p>

```owl
    <owl:AnnotationProperty rdf:about="http://purl.obolibrary.org/obo/mondo#ABBREVIATION">
        <oboInOwl:hasExactSynonym>acronym</oboInOwl:hasExactSynonym>
        <rdfs:label>acronym</rdfs:label>
        <rdfs:subPropertyOf rdf:resource="http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty"/>
    </owl:AnnotationProperty>

    <owl:Class rdf:about="http://purl.obolibrary.org/obo/DOID_0050125">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/DOID_12205"/>
        <oboInOwl:hasExactSynonym xml:lang="en">DSS</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">dengue shock syndrome</oboInOwl:hasExactSynonym>
        <rdfs:label xml:lang="en">dengue shock syndrome</rdfs:label>
    </owl:Class>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/DOID_0050125"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
        <owl:annotatedTarget xml:lang="en">DSS</owl:annotatedTarget>
        <oboInOwl:hasSynonymType rdf:resource="http://purl.obolibrary.org/obo/OMO_0003012"/>
    </owl:Axiom>
```

</p>
</details> 

## Questions
- [ ] 1. What do you guys think of using converting to `MONDO:ABBREVIATION` in the components?
- [ ] 2. If you guys want to use `MONDO:ABBREVIATION`, any idea how to fix?
  - I'm guessing this is because `OMO:0003012` is being used not as an annotation property on classes, but on axioms.

If you don't like `MONDO:ABBREVIATION` in the components or we are having difficulty converting it, I'll just add `OMO:0003012` to `properties.txt`

Reply:
I suggested to add `OMO:0003012` to properties.txt and mentioned that in the ROBOT file it would need to be mapped to `MONDO:ABBREVIATION` because I already tried a few things unsuccessfully to map `OMO:0003012` to the mondo property. It's too much overhead to deal with now so please just use `OMO:0003012`.

Reply:
If there are issues in the ROBOT template using `MONDO:ABBREVIATION` to add content to Mondo let us know!

Reply:
There's no issue using `MONDO:ABBREVIATION` in the ROBOT template. I was just thinking of doing this conversion from `OMO:0003012` earlier in the pipeline, during component creation. It isn't working though so I'll scrap. Can add it back if Nico/we know how to do it.

Reply:
Just to add this back here, there is also a discussion item for the Tech call on what property Mondo should use to represent source abbreviation/acronyms so while the Tech team plan is to use `MONDO:ABBREVIATION`, this is pending the discussion Friday.

Reply:
The change for [this line](https://github.com/monarch-initiative/mondo-ingest/pull/93/commits/39cef5c47001980fada1872695f52252312afdaa#diff-be21abb58526f37a1b7d1c133de58f149d0e5026d523a052e888d1fa42d480beR10) should be removed.

In the future, if you think there is another option that may be better please ask on Slack vs. not doing what was asked.